# -*- coding: mbcs -*-
#
# Abaqus/CAE Release 6.14-2 replay file
# Internal Version: 2014_08_22-09.00.46 134497
# Run by leal26 on Wed Mar 15 15:37:39 2017
#

# from driverUtils import executeOnCaeGraphicsStartup
# executeOnCaeGraphicsStartup()
#: Executing "onCaeGraphicsStartup()" in the site directory ...
from abaqus import *
from abaqusConstants import *
session.Viewport(name='Viewport: 1', origin=(0.0, 0.0), width=234.129150390625, 
    height=264.936126708984)
session.viewports['Viewport: 1'].makeCurrent()
session.viewports['Viewport: 1'].maximize()
from caeModules import *
from driverUtils import executeOnCaeStartup
executeOnCaeStartup()
session.viewports['Viewport: 1'].partDisplay.geometryOptions.setValues(
    referenceRepresentation=ON)
execfile('main_script.py', __main__.__dict__)
#: A new model database has been created.
#: The model "Model-1" has been created.
session.viewports['Viewport: 1'].setValues(displayedObject=None)
#* WindowsError: [Error 2] The system cannot find the file specified
#* File "main_script.py", line 65, in <module>
#*     grid_size, plot=True)
#* File "C:\Users\leal26\Documents\GitHub\leafy_wingy\venation_module.py", line 
#* 36, in generate_venation
#*     stderr=None)
#* File "C:\SIMULIA\Abaqus\6.14-2\tools\SMApy\python2.7\lib\subprocess.py", 
#* line 679, in __init__
#*     errread, errwrite)
#* File "C:\SIMULIA\Abaqus\6.14-2\tools\SMApy\python2.7\lib\subprocess.py", 
#* line 896, in _execute_child
#*     startupinfo)
